AndelaOSP/wire
# #154553173 Fix test configurations

#### What does this PR do?
This PR changes tests configurations.
#### Description of Task to be completed?
Changes tests configurations and incorporates jest alongside enzyme.
#### How should this be manually tested?
-Once tests are written. one should simply run `jest` or `npm run test:fend` or in  watch mode  `jest --watch`
#### Any background context you  want to provide?
Before this PR, enzyme was the library used for testing which lacked in some areas such that It did not have a test runner or an assertion library therefore the need of jest which comes with a test runner, assertion library, and good mocking support.
NB: All tests files should now be written under the `__tests__` folder for easier separation of concerns.
#### What are the relevant pivotal tracker stories?
[#154553173](https://www.pivotaltracker.com/n/projects/2117172)
#### Screenshots (if appropriate)
N/A
